examples/context_addition_from_filter.py

```diff
@@ -0,0 +1,33 @@
+from typing import Any, Dict, Optional, Union
+
+from aiogram import Router
+from aiogram.filters import Filter
+from aiogram.types import Message, User
+
+
+router = Router(name=__name__)
+
+
+class HelloFilter(Filter):
+    def __init__(self, name: Optional[str] = None) -> None:
+        self.name = name
+
+    async def __call__(
+        self,
+        message: Message,
+        event_from_user: User
+        # as I said previously, filters also can accept keyword parameters like in handlers
```

Comment:
"As I said previously" is not clear sentence for this place in due to example can be viewed without reading documentation and this comment is first in this file, you say nothing previously.